### ISSUE TITLE
Slider server only works on HTTPS, not on HTTP

### ISSUE THREAD

**Issue by leephillips** [JuliaPluto/PlutoSliderServer.jl#47]
The slider server works perfectly locally, but the exact same notebook and versions of everything does not work remotely.

Julia v.1.7.1
Pluto v0.17.4
PlutoSliderServer v0.3.3

Starting a remote server gets me a correctly rendered notebook in the browser, but moving a slider has no effect. There's a JavaScript error in the browser console:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'digest')
    at hash_arraybuffer (PlutoHash.js:15)
```
The line referred to is:
```
    const hashed_buffer = await window.crypto.subtle.digest("SHA-256", data)
```
This problem occurs both when starting the server with 0.0.0.0 and with 127.0.0.1 and accessing through a reverse proxy.

**Comment by leephillips** [comment]
I got this to work. I had been trying with HTTP access, as the documentation shows. But the error message suggests that HTTPS is expected. So after setting up SSL, a subdomain, and a certificate, everything works. Suggest that either the documentation be corrected or, if plain HTTP can work, that it explain how.

**Comment by pankgeorg** [comment]
This is due to the spec: https://developer.mozilla.org/en-US/docs/Web/API/Crypto, note the 'secure context' yellow label.
That means, localhost or HTTPS. It's on the browser side of things so there is not an override unfortunately.

**Comment by fonsp** [comment]
I think this is our error: we should not be using a lib that only works on HTTPS (our application of SHA256 is not a security thing), let's find a polyfill

**Comment by fonsp** [comment]
Fixed! Update Pluto to 0.17.6 or above and it will work
